app/assets/javascripts/application.js

```diff
@@ -1,6 +1,11 @@
+//= require govuk_publishing_components/dependencies
```

Comment:
This file shouldn't be included for an app that has slimmer (such as this) as it introduces the dependencies that static already provides (we'll end up bundling jQuery twice on GOV.UK for instance)

To get these dependencies in a test environment only you can take an approach like: https://github.com/alphagov/smart-answers/commit/bda1c918f6d179925f91766bce6fd8ce27a7063d